resources/views/settings/notifications/organization.blade.php

```diff
@@ -21,10 +21,12 @@
 
             <p><strong>{{ __('Name') }}:</strong> {{ $user->organization->contact_person_name }}</p>
             @if ($user->organization->contact_person_email)
-                <x-contact-point type="email" :value="$user->organization->contact_person_email" :preferred="$user->organization->preferred_contact_method === 'email'" />
+                <x-contact-point type="email" :value="$user->organization->contact_person_email" :preferred="$user->organization->preferred_contact_method === 'email' &&
+                    $user->organization->contact_person_phone" />
             @endif
             @if ($user->organization->contact_person_phone)
-                <x-contact-point type="phone" :value="$user->organization->contact_person_phone->formatForCountry('CA')" :preferred="$user->organization->preferred_contact_method === 'phone'" :vrs="$user->organization->contact_person_vrs" />
+                <x-contact-point type="phone" :value="$user->organization->contact_person_phone" :preferred="$user->organization->preferred_contact_method === 'phone' &&
+                    $user->organization->contact_person_email" :vrs="$user->organization->contact_person_vrs" />
```

Comment:
I'm assuming these values for `email` and `phone` are coming from an enum. We should probably file an issue, if one doesn't exist, to make comparisons like this against the value from the related enum. That way if it gets updated you won't have to make changes throughout the codebase again, and it identifies what the value range could be.

Reply:
They don't actually come from an Enum. If you want to file an issue that would be great.

Reply:
Discussed and the issue will suggest renaming `NotificationMethod` to `ContactMethod` and using it for both its current role and this use case.

Reply:
Filed as #1345 and #1343